packages/components/tests/dummy/app/templates/components/alert.hbs

```diff
@@ -0,0 +1,257 @@
+{{page-title "Alert components"}}
+
+<h2 class="dummy-h2">Alerts</h2>
+
+<section>
+  <h3 class="dummy-h3">Scratch reference links</h3>
+  <p>Starting with Alert: "Global" and Alert: "Page or Inline". (Alert: "Toast" coming next.)</p>
+  <ul class="dummy-paragraph">
+    <li>
+      <a href="https://docs.google.com/document/d/1MC0MTm8-0F26s6Y-gou_RsMj6rtKS6jC-eM7rmruqLY/edit#heading=h.gjdgxs">
+        Alerts Components Requirement Document (CRD)
+      </a>
+    </li>
+    <li>
+      <a
+        href="https://www.figma.com/file/noyY6dUMDYjmySpHcMjhkN/branch/Ru0h1oqy6r7SeGxwtQ6085/HDS-Product---Components-%5BWIP%5D?node-id=4023%3A15691"
+      >
+        WIP Figma file
+      </a>
+    </li>
+  </ul>
+</section>
+
+<section>
+  <h3 class="dummy-h3">Component API</h3>
+  <p class="dummy-paragraph">Here is the API for the component:</p>
+
+  <dl class="dummy-component-props">
+    <dt>type <code>enum</code></dt>
+    <dd>
+      <p>Has a border and a background color. Names
+        <code class="dummy-code">global</code>
+        and
+        <code class="dummy-code">page or inline</code>
+        are temporary and subject to change.</p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">global</li>
+        <li>page or inline</li>
+      </ol>
+    </dd>
+
+    <dt>isCompact <code>boolean</code></dt>
+    <dd>
+      <p>
+        <code class="dummy-code">isCompact</code>
+        is a variation on the
+        <code class="dummy-code">page or inline</code>
+        alert type ONLY. An
+        <code class="dummy-code">isCompact</code>
+        alert does NOT have a border. An
+        <code class="dummy-code">isCompact</code>
+        alert does NOT have a background color.
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li>true</li>
+        <li class="default">false</li>
+      </ol>
+    </dd>
+
+    <dt>intent <code>enum</code></dt>
+    <dd>
+      <p><code class="dummy-code">intent</code>
+        result in a default
+        <code class="dummy-code">icon</code>, which can be overriden.
+        <code class="dummy-code">intent</code>
+        chooses a background color, border,
+        <code class="dummy-code">titleText</code>
+        color (if applicable), and
+        <code class="dummy-code">bodyText</code>
+        color (if applicable).
+      </p>
+      <p>Acceptable values:</p>
+      <ol>
+        <li class="default">neutral</li>
+        <li>highlight</li>
+        <li>success</li>
+        <li>warning</li>
+        <li>critical</li>
+      </ol>
+    </dd>
+
+    <dt>hasIcon <code>boolean</code></dt>
```

Comment:
[question] not sure why you need an `hasIcon` prop: this can be computed by the fact that `@icon` is passed or not

Reply:
removed https://github.com/hashicorp/design-system/pull/92/commits/e4f4b7739c4a386e06a0bc2a7c5492fe57416251